spec/relation-entities/relation-entities-read.spec.ts

```diff
@@ -190,7 +190,7 @@ describe('Relation Entities Read', () => {
 
         const { body: commentListBodyNext } = await request(app.getHttpServer())
             .get('/comment')
-            .query({ nextCursor: commentListBody.metadata.nextCursor })
+            .query({ nextCursor: commentListBody.metadata.query })
```

Comment:
possibly typo? It seems that `nextCursour` should be used rather than `query`

Reply:
That's not a typo.

Within the service logic, query and nextCursor are still managed separately.
It's just that query has been removed from the interface.